### PR TITLE
Add extraction/re-packing of rootfs

### PR DESCRIPTION
**Pull request by DeonMarais64**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,8 +35,11 @@ mv Linux zImage
 mv Ramdisk rootfs.cpio.gz
 ```
 
-Now we can replace some of these files as required with our modifications, and
-build the FIT image and `.frm` file as described in the
+Now we can replace some of these files as required with our modifications. In 
+order to modify the rootfs, is is necessary to extract it and re-pack it as 
+indicated 
+[`below`](https://github.com/daniestevez/pluto-firmware-modifications/edit/main/README.md#extraction-and-re-packing-of-the-rootfs)
```
Comment (daniestevez) [inline README.md]: This link points to the "edit" page. The correct link would be https://github.com/daniestevez/pluto-firmware-modifications/blob/main/README.md#extraction-of-the-pluto-firmware-image

I don't know if it's possible to include a relative link to a section, as in `[below](#extraction-and-re-packing-of-the-rootfs)`.

**Review by daniestevez — CHANGES_REQUESTED** [review]
Thanks. I've spotted a small problem with a link. Otherwise this is ready to go.

By the way, there was no need to close #2 and open a new pull request. You can add additional commits or force-push to the branch of a pull request.

**Comment by DeonMarais64** [comment]
The git review process and PR updates resulting from it is something that I am yet to master. Please let me know if I need to do anything further.

**Review by daniestevez — APPROVED** [review]
Thank you! Now this looks good.

No worries. It gets some time to get used to all the features of Github and the usage of git. I realized that there is a Github feature to suggest modifications that I could have used to make this process easier.

**Pull request merged**